evergreen/staticfiles/css/styles.css

```diff
@@ -118,14 +112,49 @@ footer {
     margin-left: 5%;
 }
 
+.likeButton button {
+    padding: 8px 16px;
+    border: 1px solid #4caf50;
+    border-radius: 4px;
+    cursor: pointer;
+    background-color: white;
+    color: #4caf50;
+    transition: all 0.3s ease;
+}
+
+.likeButton button.liked {
+    background-color: #4caf50;
+    color: white;
+}
+
+.likers-display {
+    font-size: 0.9em;
+    color: #666;
+    margin: 5px 0;
+}
```

Comment:
I like these styles. Doesn't fit too well with the rest of the site, but that's more a problem with the rest of the site :+1: